tests/Unleash.Tests/MockedUnleashSettings.cs

```diff
@@ -8,10 +8,10 @@ namespace Unleash.Tests
 {
     public class MockedUnleashSettings : UnleashSettings
     {
-        public MockedUnleashSettings(bool mockFileSystem = true)
+        public MockedUnleashSettings(bool mockFileSystem = true, string instanceTag = "test instance 1")
         {
             AppName = "test";
-            InstanceTag = "test instance 1";
+            InstanceTag = instanceTag;
```

Comment:
This keeps failing when used in multiple test files being run simultaneously

Reply:
That seems like a big change though. Any idea why?

Reply:
Okay we chatted, sounds like this was a legacy problem